### ISSUE TITLE
PA Review: Servers

### ISSUE THREAD

**Issue by rich-ma** [rich-ma/disunity#3]
Users can create, and join severs by name.

Seeded server names:
App Academy
DoUKNowdaWae

**Comment by d-webs** [comment]
Everything is looking awesome, Richard. A couple things:

- [x] After I create a server, I get redirected to `/servers/undefined`. 
- [ ] When I upload an image, shouldn't the little blue circle turn into a preview of the image?
- [x] Pet peeve is when errors move the rest of the page down. In your `create-server-errors` div, I recommend giving it a min height (I'd say somewhere around `25px`, but feel it out) so that there is always a small space for errors, and they do not move the modal content down when present. Same thing for joining a server
- [x] I tried to join App Academy - apparently Demo user is already in that. I then tried to join DoUKNowdaWae, and it appeared successful but nothing changed, and no new servers showed up on the sidebar. What is the expected behavior here? 
- [ ] Logout button? 

Keep up the great work!
